sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -481,7 +481,7 @@ object GpuWindowExec {
   def isBatchedRunningFunc(func: Expression, spec: GpuWindowSpecDefinition): Boolean = {
     val isSpecOkay = isRunningWindow(spec)
     val isFuncOkay = func match {
-      case _: GpuBatchedRunningWindowWithFixer => true
+      case f: GpuBatchedRunningWindowWithFixer => f.newFixer().isDefined
       case GpuAggregateExpression(_: GpuBatchedRunningWindowWithFixer, _, _, _ , _) => true
```

Comment:
Don't we need to capture the inner GpuBatchedRunningWindowWithFixer and check it here too?

Reply:
D'oh! Will fix.